src/ini.pest

```diff
@@ -1,13 +1,15 @@
-WHITESPACE = _{ " " }
-char = { ASCII_ALPHANUMERIC | "." | "_" | "/" }
+ini = { SOI ~ ((pair | comment)? ~ (NEWLINE | EOI))* ~ section* }
+WHITESPACE = _{ " " | "\t" }
+
+// section
+section = { "[" ~ name ~ "]" ~ body }
 name = @{ char+ }
-value = @{ char* }
+char = { ASCII_ALPHANUMERIC | "." | "_" | "/" }
+body = { (NEWLINE ~ (pair | comment)?)* }
 
-section = { "[" ~ name ~ "]" }
-property = { name ~ "=" ~ value }
+// key-value pair
+pair = { name ~ "="? ~ value }
```

Comment:
`"="?` will support a key-value pair without delimiting equal-sign:

```
the_key the_value
```

I guess that's probably not what you wanted to achieve, are you trying to match keys without value? If so, is that a feature of any INI-implementation?

Reply:
`the_key the_value` would actually be just one very long key. Yes, that's what I was trying to support, as the `=` was superfluous from a parsing perspective. I never said anything about a space as a delimiter.

And no, I don't think it's an existing feature.